src/javascript/app_2/App/Components/Elements/FullPageModal/full_page_modal.jsx

```diff
@@ -0,0 +1,51 @@
+import React       from 'react';
+import PropTypes   from 'prop-types';
+
+const FullPageModal = ({
+    body,
+    cancel_button_text,
+    confirm_button_text,
+    onConfirm,
+    onCancel,
+    show,
+    title,
+}) => {
+    if (show) {
+        return (
+            <div className='full-page-modal'>
+                <div className='modal-dialog'>
+                    <h1>{title}</h1>
+                    <p>{body}</p>
+                    <div className='modal-footer'>
+                        <div
+                            className='btn flat effect primary'
+                            onClick={onCancel}
+                        >
+                            <span>{cancel_button_text}</span>
+                        </div>
+                        <div
+                            className='btn flat effect primary'
+                            onClick={() => onConfirm()}
+                        >
+                            <span>{confirm_button_text}</span>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        );
+    }
+
+    return <React.Fragment />;
+};
+
+FullPageModal.propTypes = {
+    body               : PropTypes.string,
+    cancel_button_text : PropTypes.string,
+    confirm_button_text: PropTypes.string,
+    onCancel           : PropTypes.func,
+    onConfirm          : PropTypes.func,
+    show               : PropTypes.bool,
```

Comment:
@easteregg 
Would you plz follow the [naming conventions style guide](https://github.com/binary-com/binary-static/tree/new-app/src/javascript#naming-conventions-boolean) in order to name boolean variables?